openapi-mock/server/mock_controller.js

```diff
@@ -7,7 +7,10 @@ const pretty_yaml = require('json-to-pretty-yaml');
 const util = require('util');
 const LOGGER = require("./logger").logger
 var morgan = require('morgan');
-
+const DIR_NAME = "./generated/";

```

Comment:
Please add to Makefile.clean goal the deletion of the folder

Reply:
Done